### PR TITLE
added Anturnia

### PR DESCRIPTION
**Pull request by FrinshHD**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by JakobBruening — CHANGES_REQUESTED** [review]
`icon@2x.png` is required.

File: minecraft_servers/anturnia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "kitpvp": {
+      "name": "KitPVP [1.8/1.19]",
+      "command": "/kitpvp",
+      "color": "#00aa00"
+    }
+  },
+  "discord": {
+    "server_id": 870227603615805441,
+    "rename_to_minecraft_name": true
```
[the remark this answers is not in the document]
Comment (FrinshHD) [reply minecraft_servers/anturnia/manifest.json]: I changed it now and i am sorry

**Review by JakobBruening — APPROVED** [review]
Thank you 👍🏼

**Comment by FrinshHD** [comment]
When can i see the changes?

**Pull request merged**